Princesso/Send-IT
# Implement Week2 Feedback

### What this Pull Request does
This **PR** 

- Removes SQL queries from routes folder and adds them to a controllers directory
- Adds a helper class
- Adds a .eslintrc file
- Adds .env to .gitignore
- Moves badges to top of README
- Adds all .DS_STORE into .gitignore
-Adds a database config file
- Updates SQL queries
- Updates Authentication and authorization
- Generally refactors some files
### How to Test this
- Visit the README file to see where it is hosted on heroku
- Using Postman test out API endpoints
- Add Authorization bearer to header to access routes
